### PR TITLE
Remove defaultProps

### PR DESCRIPTION
**Pull request by danielfrg**
Fixes https://github.com/AaronCCWong/react-card-flip/issues/119

### CODE REVIEW

**Review by AaronCCWong — DISMISSED** [review]
Awesome! Thanks for putting this together!

File: src/ReactCardFlip.tsx [hunk 1]
```diff
@@ -16,14 +16,28 @@ const ReactCardFlip: React.FC<ReactFlipCardProps> = (props) => {
     flipSpeedFrontToBack,
     flipSpeedBackToFront,
     infinite,
-  } = props;
-
-  const [isFlipped, setFlipped] = useState(props.isFlipped);
+    isFlipped,
```
Comment (AaronCCWong) [inline src/ReactCardFlip.tsx:19]: This conflicts with the internal state variable defined on line 35.

**Review by AaronCCWong — CHANGES_REQUESTED** [review]

File: src/ReactCardFlip.tsx (hunk 1 above)
Comment (danielfrg) [reply src/ReactCardFlip.tsx:19]: I think it should be fixed now.

**Review by AaronCCWong — APPROVED** [review]

**Comment by danielfrg** [comment]
Thanks for the lib and the quick new release! Very helpful!

**Comment by danielfrg** [comment]
For some reason I still hit https://github.com/AaronCCWong/react-card-flip/issues/119 after updating. Maybe you forgot to run `npm run build` to update the compiled code?

**Comment by AaronCCWong** [comment]
Thanks for pointing this out! Updated

**Comment by danielfrg** [comment]
Thank you!

**Pull request merged**